ProjectName.cmake

```diff
@@ -1,6 +1,6 @@
 SET(PROJECT_NAME Trilinos)
 
-SET(${PROJECT_NAME}_ENABLE_SECONDARY_TESTED_CODE_DEFAULT  ON)
+SET(${PROJECT_NAME}_ENABLE_SECONDARY_TESTED_CODE_DEFAULT  OFF)
```

Comment:
This is a break in backward compatibility.  This will break many users configure scripts.

Reply:
Agreed.  This is the most dangerous part of this change (and is what the large text in the description is intended to convey).